dlc_developer_config.toml

```diff
@@ -28,7 +28,7 @@ benchmark_mode = false
 [build]
```

Comment:
I think you will need to use the updated dlc_developer_config.toml file to run the PR tests with the recent CI/CD changes.